pasqal_cloud/client.py

```diff
@@ -108,18 +110,27 @@ def _request(
         payload: Optional[Union[Mapping, Sequence[Mapping]]] = None,
         params: Optional[Mapping[str, Any]] = None,
     ) -> JSendPayload:
-        rsp = requests.request(
-            method,
-            url,
-            json=payload,
-            timeout=TIMEOUT,
-            headers={"content-type": "application/json"},
-            auth=self.authenticator,
-            params=params,
-        )
-        rsp.raise_for_status()
-        data: JSendPayload = rsp.json()
-
+        successful_request: bool = False
+        iteration: int = 0
+        while not successful_request and iteration <= HTTP_RETRIES:
```

Comment:
Looks like I forgot to update `successful_request` when the request returns a payload

Reply:
This is still missing AFAICT

Reply:
I know? I've not asked for a re review  yet

Reply:
Ok sure. Just made a pass to checkout your answers to my comments as I hadn't done so already :)